docs/quick-start.md

```diff
@@ -11,7 +11,10 @@ gRPC requires Swift 5.0 or higher.
 #### Install Protocol Buffers v3
 
 Install the protoc compiler that is used to generate gRPC service code. The
-simplest way to do this is to download pre-compiled binaries for your
+simplest way to do this is to use Homebrew.
+```brew install swift-protobuf```
```

Comment:
This section is about install `protoc` rather than the plugins, so this should be `protobuf` instead of `swift-protobuf`:

```sh
$ brew install protobuf
```